deployment/templates/fluentd.yaml

```diff
@@ -165,3 +167,7 @@ parameters:
   description: "Location of CA cert for validating connectiong to ElasticSearch to write cluster logs"
   name: OPS_CA
   value: "/etc/fluent/keys/ca"
+-
+  description: "JSON format configuration for log throttle management"
+  name: THROTTLE_SETTINGS
```

Comment:
What would be setting this - someone manually editing the fluentd DC/daemonset?

I don't really like to put JSON inside a string or in an env var... too many ways to muff that and it's unreadable. Is there any way to just have this specify a filename to read, and have that default to a file in the fluentd secret, and have the generator just use the default if that file's not there?